scripts-available/CDB_Overviews.sql

```diff
@@ -140,16 +166,15 @@ RETURNS TABLE(base_table REGCLASS, z integer, overview_table REGCLASS)
 AS $$
   DECLARE
     schema_name TEXT;
-    table_name TEXT;
+    _table_name TEXT;
   BEGIN
-    -- TODO: review implementation of CDB_UserTables an suitability for this
-    SELECT * FROM _cdb_split_table_name(reloid) INTO schema_name, table_name;
+    SELECT * FROM _cdb_split_table_name(reloid) INTO schema_name, _table_name;
     RETURN QUERY SELECT
       reloid AS base_table,
-      _CDB_OverviewTableZ(cdb_usertables) AS z,
-      ('"' || schema_name|| '"."' ||cdb_usertables || '"')::regclass AS overview_table
-      FROM CDB_UserTables()
-      WHERE _CDB_IsOverviewTableOf((SELECT relname FROM pg_class WHERE oid=reloid), cdb_usertables)
+      _CDB_OverviewTableZ(table_name) AS z,
```

Comment:
Not a bug but confusing naming: the one with underscore is variable used to get the second result of `_cdb_split_table_name` which isn't actually used; the other identifer is a column of `_CDB_UserTablesInSchema`. Should change the name of the first to make more evident it not being used.